### DISCUSSION TITLE
Circular import error

### DISCUSSION THREAD

**Discussion by Ashwanth369** [apache/airflow#20750] [General]
### Apache Airflow version

2.2.2

### What happened

I am working in building a project on top of airflow. One of the files uses the datetime.py file under the operators directory. It seems the file has this line:

**import datetime**

and because of this, I am encountering the circular import error:

**AttributeError: partially initialized module 'datetime' has no attribute 'date' (most likely due to a circular import).**

I am also facing a similar issue because of the sqlalchemy.py file under the utils folder. Is there anyway to overcome this issue?

### Operating System

Ubuntu 20.04

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Converted it to discussion - this is not an issue of airflow. if you would like to get help with it - I think you must post a bit more details - like your code that generates the import (but I would guess it's just your sequence of imports that is wrong).
